pygenometracks/plotTracks.py

```diff
@@ -275,6 +276,10 @@ def main(args=None):
                      track_label_width=args.trackLabelFraction,
                      plot_regions=regions, plot_width=args.plotWidth)
 
+    # Create dir if dir does not exists:
+    # Modified from https://stackoverflow.com/questions/12517451/automatically-creating-directories-with-file-output
+    os.makedirs(os.path.dirname(os.path.abspath(args.outFileName)), exist_ok=True)
```

Comment:
https://docs.python.org/3/library/os.html#os.makedirs

What does `confused` mean? `[makedirs()](https://docs.python.org/3/library/os.html#os.makedirs) will become confused if the path elements to create include [pardir](https://docs.python.org/3/library/os.html#os.pardir) (eg. “..” on UNIX systems).`

Reply:
I will make a test. With an output which contains `../` or `./`. And come back to you.

Reply:
It works. `os.path.abspath` resolves any `..` or `.`.

Reply:
I don't know what 'confused' mean because I tried:
```
>>> os.makedirs("../toto")
>>> os.makedirs("pygenometracks/../toto")
>>> os.makedirs("pygenometracks/../toto/tralal/../t")
```
And all worked. Maybe the last one got an unexpected behaviour as it created both toto/tralal and toto/t.